projects/angular/components/ui-grid/src/ui-grid.component.ts

```diff
@@ -991,6 +993,18 @@ export class UiGridComponent<T extends IGridDataEntry> extends ResizableGrid<T>
         }
     }
 
+    searchableDropdownValue(searchableDropdown: UiGridSearchFilterDirective<T>): ISuggestValue[] {
+        if (searchableDropdown.value) {
+            if (searchableDropdown.multiple) {
+                return searchableDropdown.value! as ISuggestValue[];
+            } else {
+                return [searchableDropdown.value! as ISuggestValue];
```

Comment:
doesn't it infer not nullable?
```suggestion
                return searchableDropdown.value as ISuggestValue[];
            } else {
                return [searchableDropdown.value as ISuggestValue];
```

Reply:
It does indeed. Made the change